templates/zerver/emails/new_login/new_login_alert.txt

```diff
@@ -0,0 +1,18 @@
+{# Informs user of a new login to their account #}
+
+Hello, {{ user.short_name | title }}.
```

Comment:
I think this should be `{{ user.full_name }}` -- `short_name` isn't set effectively in Zulip right now